Misc/NEWS.d/3.12.0a1.rst

```diff
@@ -3561,8 +3561,8 @@ with :func:`os.pidfd_open` in non-blocking mode. Patch by Kumar Aditya.
 .. nonce: mkYl5q
 .. section: Library
 
-Implement Enum __contains__ that returns True or False to replace the
-deprecated behaviour that would sometimes raise a TypeError.
+Implement ``Enum.__contains__`` that returns ``True`` or ``False`` to replace the
+deprecated behaviour that would sometimes raise a ``TypeError``.
```

Comment:
```suggestion
deprecated behaviour that would sometimes raise a :exc:`TypeError`.
```

Reply:
I don't think there is any value in turning TypeError to a link in this context.

Reply:
I generally prefer to assume close to no knowledge on the part of the reader when it comes to the CPython docs. But I don't have a strong opinion here.

Reply:
I think there's enough value to keep the link.